src/components/ExcelImportDialog.tsx

```
@@ -0,0 +1,479 @@
+  CardContent,
+  List,
+  ListItem,
+  ListItemIcon,
+  ListItemText,
+  Fade,
+  Collapse
+} from '@mui/material';
+import {
+  CloudUpload,
+  CheckCircle,
+  Error,
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)